src/common/log/parser.py

```diff
@@ -192,19 +193,53 @@ def parse_fmu_xml_log(filename, modulename = 'Model', accept_errors=False):
             print('Parsed log will be incomplete')
         else:
             raise Exception('Failed to parse XML FMU log:\n' + repr(e))
-    
+
     return handler.get_root()
 
-def extract_xml_log(destfilename, filename, modulename = 'Model'):
+def extract_xml_log(dest, log, modulename = 'Model'):
     """
-    Extract the XML contents of a FMU log and write as a new file destfilename.
-
-    modulename selects the module as recorded in the beginning of each line by
+    Extract the XML contents of a FMU log and write as a new file dest, or extract into a stream.
+    Input argument 'modulename' selects the module as recorded in the beginning of each line by
     FMI Library.
+
+        dest::
+
+            file_name  --
```

Comment:
I think file_name should be removed right? It should say only dest?

Reply:
Correct this is just an error due to search/replace. I've now fixed this.